rpm/containerd.spec

```diff
@@ -121,8 +121,8 @@ make -C /go/src/github.com/opencontainers/runc BINDIR=%{_topdir}/BUILD/bin BUILD
 
 %install
 cd %{_topdir}/BUILD
-mkdir -p %{buildroot}%{_bindir}
-install -D -m 0755 bin/* %{buildroot}%{_bindir}
+mkdir -p %{buildroot}%{_exec_prefix}/local/bin
+install -D -m 0755 bin/* %{buildroot}%{_exec_prefix}/local/bin
```

Comment:
Unfortunately looks like there's no RPM macro for `/usr/local/bin`  `%{_exec_prefix}` (or `%{_prefix}`  not sure which one was preferred ?) came closest https://docs.fedoraproject.org/en-US/packaging-guidelines/RPMMacros/#_macros_for_paths_set_and_used_by_build_systems

Reply:
That's probably because RPMs aren't supposed to touch it per the FHS :sweat_smile: